backend/tracim_backend/config.py

```diff
@@ -1154,6 +1155,12 @@ def _check_global_config_validity(self) -> None:
             self.check_directory_path_param(
                 "SESSION__DATA_DIR", self.SESSION__DATA_DIR, writable=True
             )
+        elif self.SESSION__TYPE in ["ext:rediscluster"]:
+            self.check_mandatory_param(
+                "SESSION__URLS",
+                self.SESSION__URLS,
+                when_str="if session type is {}".format(self.SESSION__TYPE),
+            )
```

Comment:
can we check SESSION_URL is empty (if it is mandatory when SESSION_URLS is set)

Reply:
SESSION_URL doesn't have to be empty. I juste mimiced actual behavior and description.